lib/src/modal_page/sliver_wolt_modal_sheet_page.dart

```diff
@@ -233,6 +233,12 @@ class SliverWoltModalSheetPage {
   /// notch and system gesture areas.
   final bool? useSafeArea;
 
+  /// Controls the draggability of the bottom sheet content. This setting overrides the value provided
+  /// via [WoltModalSheet.show] specifically for this page when the modal is displayed as a bottom sheet.
+  final bool? contentDraggable;
```

Comment:
We already have ways of defining `enableDrag` in `wolt_modal_sheet.dart`

https://github.com/woltapp/wolt_modal_sheet/blob/main/lib/src/wolt_modal_sheet.dart#L328

Reply:
What I was actually thinking of was to control the `content scroll ` and the `top drag widget` separately. Since we are a package, I think we can give this flexibility. Not everyone wants the page to close when the list view is dragged down. They may only want it to close when dragged from the top. This is my point of view, of course, we can discuss it :)

Reply:
As discussed in private, let's wait for user feedback. I am afraid of comlicating the API. Probably we should have never ever exposed the main content scroll physics.